File: backend/src/main/java/com/example/backend/dto/AdvertisementEditDto.java
package com.example.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AdvertisementEditDto {
    private byte[] photo;
    private String name;
    private String description;
    private Boolean statusActive;
    private int category;
}

File: app/src/main/java/com/example/dropletbarterapp/mainscreens/advertisements/dto/AdvertisementEditDto.kt
package com.example.dropletbarterapp.mainscreens.advertisements.dto

import com.google.gson.annotations.SerializedName

class AdvertisementEditDto(
    @SerializedName("photo") val photo: ByteArray,
    @SerializedName("name") val name: String? = null,
    @SerializedName("description") val description: String? = null,
    @SerializedName("statusActive") val statusActive: Boolean? = null,
    @SerializedName("category") val category: Int,
    @SerializedName("ownerId") val ownerId: Long? = null,
)